ddtrace/tracer/option.go

```diff
@@ -482,6 +554,43 @@ func (c *config) loadAgentFeatures() {
 	}
 }
 
+// ImportIntegration labels the given integration as imported
+func ImportIntegration(ip string) bool {
```

Comment:
Reading this again I think this name confused me a bit since I think it could be read as doing some sort of "Import" work for the integration. Maybe a name like `MarkIntegrationImported`?  Also just seeing that the param is `ip` is that short for something?